addons/website_event_booth/views/event_booth_templates.xml

```diff
@@ -22,22 +22,38 @@
                         </div>
                     </div>
                 </section>
-                <div t-if="event.is_finished" class="container d-flex flex-column flex-grow-1">
-                    <span class="alert alert-info">This event is finished. It's no longer possible to book a booth.</span>
+                <div t-if="event.is_finished">
+                    <div class="col-12">
```

Comment:
You have a `col-x` item that's not a `.row` direct descendant.
Either `col-12` is unnecessary or a `.row` is missing. Could you have a look?